### PR TITLE
Dracut execution has to be done after clevis binds the disk

### PR DESCRIPTION
**Pull request by kokesak**
Dracut execution has to be done after clevis binds the disk

### CODE REVIEW

**Review by sarroutbi — APPROVED** [review]
Change LGTM

**Review by sarroutbi — APPROVED** [review]
Minor comments. Change LGTM

File: Sanity/pkcs11/single-encrypted-disk/runtest.sh [hunk 1]
```diff
@@ -76,8 +76,9 @@ PACKAGE="clevis"
 #      load the disk during the boot time
 #   4. Create a new initramfs that contains softhsm libraries and tokens so it
 #      can be picked up by Clevis during the boot time
-#   5. Use the TPM2 and the softhsm token as a 2 factor clevis encryption of the disk
-#   6. Reboot the system and verify that the disk was successfully decrypted by Clevis
+#   5. Clevis bind the disk using the pkcs11 softhsm token
```
[the remark this answers is not in the document]
Comment (kokesak) [reply Sanity/pkcs11/single-encrypted-disk/runtest.sh:79]: Thanks, I'll change it after the tests finishes!

[the remark this answers is not in the document]
Comment (kokesak) [reply Sanity/pkcs11/single-encrypted-disk/runtest.sh]: done

**Pull request merged**